extensions/dataloading/dataloading-spi/src/test/java/org/eclipse/dataspaceconnector/dataloading/DataLoaderTest.java

```diff
@@ -48,17 +48,15 @@ void setUp() {
     @Test
     void insert() {
         var te = new TestEntity("Test Desc", 3);
-        sinkMock.accept(te);
-        expectLastCall();
-        replay(sinkMock);
+
         dataLoader.insert(te);
-        verify(sinkMock);
+
+        verify(sinkMock).accept(te);
```

Comment:
adding `verifyNoMoreInteractions` would resemble more closely the behaviour of a `strictMock`